deploy/k8s/helm/deploy-all.sh

```diff
@@ -160,7 +160,7 @@ ingress_values_file="ingress_values.yaml"
 
 if [[ $use_local_k8s ]]; then
   ingress_values_file="ingress_values_dockerk8s.yaml"
-  dns="localhost"
+  dns="host.minikube.internal"
```

Comment:
Though as of now, only Docker Desktop is only the supported environment, it might not be a good idea to hard code the DNS name to any specific tool.